entrypoint.sh

```diff
@@ -140,7 +140,15 @@ if [ "$INPUT_DOCX" = "true" ]; then
   echo "::endgroup::"
 fi
 
+# TODO evaluate to move this entire linting block before any document generation.
 if [ "$INPUT_LINT" = "true" ]; then
+  echo "::group::Linting Markdown"
+  # Enabled checks:
```

Comment:
Why would we only lint list and nothing else?

Reply:
This is the only `mdl` linting rule I've tested. I think additional rules should be enabled on demand.